### PR TITLE
Add tutorials for prediction markets and update color scheme

### PR DESCRIPTION
**Pull request by maltekliemann**
Second batch of commits of the documentation overhaul. Features slightly new color scheme that works with (pale) dark and light mode, and a major rewrite of the "Prediction Markets" and "Using Zeitgeist Markets" chapters, as well as a new chapter on Liquidity Pools, which seem to have caused many users trouble during the beta.

I've also reworked some of the info on the Zeitgeist Beta.

### CODE REVIEW

**Comment by maltekliemann** [comment]
Oh yeah, there's also a new landing page which is work in progress. I'd like to have a landing page like polkadot-wiki, but we're still searching for a react dev in the Seer Program.

**Review by sea212 — COMMENTED** [review]
Thanks for this PR maltekliemann. It looks better now and contains much more information, well done.

File: docs/battery_station.mdx [hunk 1]
```diff
@@ -3,11 +3,9 @@ id: battery-station
 title: Battery Station
 ---
 
-import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
 
-:::important Zeitgeist Battery Station is a test network in beta phase.
-You can
+:::important Zeitgeist Battery Station is a test network in beta phase. You can
```
[the remark this answers is not in the document]
Comment (maltekliemann) [reply docs/battery_station.mdx]: My bad about the info boxes. They're now protected from the linter, but now you're stuck formatting them manually (you can format selected parts using prettier in some editors). Still, this is probably the way to go right now.

Also, you need a new `prettier-ignore` comment for every paragraph in the info box.

File: docs/using-zeitgeist-markets.md [hunk 2]
```diff
@@ -1,124 +1,348 @@
...
+slashed.
+
+## Advanced Topics
+
+### Arbitrage on Zeitgeist
+
+Zeitgeist's AMM, the Rikiddo scoring rule, does not guarantee that for any
+market the sum of the prices of all outcome assets is approximately equal to 1
+ZTG, or that the price of any outcome tokens remains below 1 ZTG. High prices
+are indicators of low liquidity or a volatile market.
```
[the remark this answers is not in the document]
Comment (maltekliemann) [reply docs/using-zeitgeist-markets.md:305]: Thanks for pointing that out. I'll make some corrections in a minute. However, I don't think that the price of each individual asset is bounded by 1 ZTG. I remember very clearly that some assets went over 1ZBS when I did a rug pull on one of the markets, so I went back and checked:

Screenshot 2021-11-22 at 14 37 33

I believe others have observed similar things in other markets. Am I missing something?

**Comment by lsaether** [comment]
Thanks @maltekliemann ! The content changes look good. However, I would recommend creating a separate PR for the landing page while it is still WIP and reverting this PR back to using "Getting Started" as index. That way we can merge this PR without requiring the landing page complete.

**Comment by maltekliemann** [comment]
@lsaether Done!

**Review by lsaether — APPROVED** [review]

**Pull request merged**